doc/oper.xml

```diff
@@ -1387,3 +1387,27 @@ gap> DigraphShortestDistance(gr, [1, 3], [3, 5]);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphClosure">
+<ManSection>
+  <Oper Name="DigraphClosure" Label="for a digraph and positive integer"
+        Arg="digraph, k"/>
+  <Returns>A digraph</Returns>
+  <Description>
+    Given a symmetric loopless digraph with no multiple edges <A>digraph</A>,
+    the <A>k</A>-closure of <A>digraph</A> is defined to be the unique smallest
+    symmetric loopless digraph <C>C</C> with no multiple edges on the vertices
+    of <A>digraph</A> which contains all the edges of <A>digraph</A> and
+    satsifies the property that the sum degrees of every two non-adjacenct
+    vertices in <C>C</C> is less than <A>k</A>.<P/> 
+
+    The function returns the <A>k</A>-closure of <A>digraph</A>.
```

Comment:
Should be either `This operation` (not "the"), or `The operation <C>DigraphClosure</C>`.